dataGenerator/dbLoader.js

```diff
@@ -0,0 +1,113 @@
+const pg = require('pg');
+const moment = require('moment');
+
+const connectionString = "postgres://noahr@localhost/bsh-rrdb";
+const pgClient = new pg.Client(connectionString);
+pgClient.connect();
+
+let loadStart = moment();
+
+let promises = [];
+// Clear data in tables
+promises.push(pgClient.query(`
+TRUNCATE TABLE reviews CASCADE
+`))
+promises.push(pgClient.query(`
+TRUNCATE TABLE chars CASCADE
+`))
+promises.push(pgClient.query(`
+TRUNCATE TABLE images
+`))
+promises.push(pgClient.query(`
+TRUNCATE TABLE reviews_chars
+`))
+
+// Copy Data from CSV files to database
+promises.push(pgClient.query(`
+COPY reviews(id,product_id,summary,body,rating,name,email,date,recommend,helpfulness,response,reported)
+FROM '${__dirname}/generatedReviews.csv'
+DELIMITER ','
+CSV HEADER;
+`))
+promises.push(pgClient.query(`
+COPY chars(id,char_name)
+FROM '${__dirname}/generatedChars.csv'
+DELIMITER ','
+CSV HEADER;
+`))
+promises.push(pgClient.query(`
+COPY images(id,url,review_id)
+FROM '${__dirname}/generatedImages.csv'
+DELIMITER ','
+CSV HEADER;
+`))
+promises.push(pgClient.query(`
+COPY reviews_chars(id,review_id,char_id,value)
+FROM '${__dirname}/generatedRevsChars.csv'
+DELIMITER ','
+CSV HEADER;
+`))
+
+
+Promise.all(promises)
```

Comment:
Promise.all seems like a great way to do this without having to wait for each one to complete